packages/opentelemetry-node-tracer/src/NodeTracer.ts

```diff
@@ -24,10 +63,12 @@ export class NodeTracer extends BasicTracer {
   /**
    * Constructs a new Tracer instance.
    */
-  constructor(config: BasicTracerConfig) {
-    super(
-      Object.assign({}, { scopeManager: new AsyncHooksScopeManager() }, config)
-    );
+  constructor(config: NodeTracerConfig) {
```

Comment:
If we keep all the `NodeTracerConfig ` attributes optional, do you think this `config` also needs to be optional param?

Reply:
I would prefer not for now since it might changes but before releasing the sdk in beta or something like this we do that yeah

Reply:
Ok makes sense to me, Could you pls open an issue so that we don't miss this in the future.

Reply:
https://github.com/open-telemetry/opentelemetry-js/issues/236